src/app/_models/container/mount/mount.model.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Interface for Mount belonging to Container
+ *
+ * @interface
+ */
+export interface Mount {
+    Name: string;
```

Comment:
Unexpected upper case property names. Lower case is preferred.
See [style-03-03](https://angular.io/guide/styleguide#style-03-03) and [style-03-04](https://angular.io/guide/styleguide#style-03-04)

Reply:
As far as I know the property names are required to be the same as the json that its parsing. So I used uppercase due to the api returning uppercase property names.

Reply:
I did not know that, then it makes sense. Thank you for clarifying that.